src/Core/Usecase/Post/Export.php

```diff
@@ -11,69 +11,153 @@
 
 namespace Ushahidi\Core\Usecase\Post;
 
+use Ushahidi\App\Repository\ExportJobRepository;
+use Ushahidi\App\Repository\Form\AttributeRepository;
+use Ushahidi\App\Repository\Post\ExportRepository;
+use Ushahidi\Core\Tool\FormatterTrait;
 use Ushahidi\Core\Usecase\SearchUsecase;
 
 class Export extends SearchUsecase
 {
+	private $payload;
+	private $session;
+	private $data;
+	private $postExportRepository;
+	private $exportJobRepository;
+	private $formAttributeRepository;
+
+
+	// provides setFormatter to assign the formatter the usecase will use
+	use FormatterTrait;
+
 	// - VerifyParentLoaded for checking that the parent exists
 	use VerifyParentLoaded;
 
+	public function setExportJobRepository(ExportJobRepository $repo)
+    {
```

Comment:
something weird with this indent or is it github?

Reply:
I clearly need to fix the linters indenting and reformat all the code to match. We have mixed spaces and tabs in too many places